docs/index.md

```diff
@@ -130,3 +130,36 @@ napari is extensible! Find plugins, or develop your own!
 :::
 
 ::::
+
+## funding
+
```

Comment:
I would love to see a direct ask for support similar to conda-forge.

Many folks do not understand that a fiscally sponsored project does not mean the sponsor fully funds the project. Let's be explicit that more donations are needed.

[conda-forge home page](https://conda-forge.org/)

<img width="2294" height="1400" alt="Image" src="https://github.com/user-attachments/assets/1c19da36-83c1-4dcb-87be-da0ad712df66" />

[Donation page](https://opencollective.com/conda-forge)

Reply:
totally. I think most people have a colloquial understanding and "fiscal sponsor" seems like someone how is financially supporting something.